### PR TITLE
Support multi-arch prebuilds for M1 (Apple silicon)

### PR DESCRIPTION
**Pull request by vweevers**
Ref https://github.com/prebuild/prebuildify/issues/52

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -119,6 +122,53 @@ test('match and sort tags', function (t) {
...
 })
 
+test('parse tuples', function (t) {
+  t.same(parseTuple('linux-arm64'), {
+    name: 'linux-arm64',
+    platform: 'linux',
+    architectures: ['arm64']
+  })
+
+  t.same(parseTuple('darwin-x64_arm64'), {
```
Comment (mafintosh) [inline test.js:132]: wdyt about using + instead _? darwin-x64+arm64

Comment (vweevers) [reply test.js:132]: I suppose `+` is less likely to cause issues, because there's prior art for arch names that contain `_`, like `x86_64`? SGTM

Comment (mafintosh) [reply test.js]: 👍

**Review by mafintosh — APPROVED** [review]

**Comment by vweevers** [comment]
4.3.0

**Pull request merged**